src/webview/utils/DOMUtils.ts

```
@@ -0,0 +1,127 @@
+    }
+    
+    return element;
+  }
+
+  /**
+   * イベントリスナーを安全に追加
+   */
+  public static addEventListenerSafe<K extends keyof HTMLElementEventMap>(
+    element: HTMLElement | null,
+    type: K,
+    listener: (this: HTMLElement, ev: HTMLElementEventMap[K]) => any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)